releases/compiler/parallel.go

```diff
@@ -30,19 +28,14 @@ func New(cfg Config) *Parallel {
 	return &Parallel{
 		compiler:    newCompiler(cfg.BaseDir, cfg.LDFlags),
 		parallelism: cfg.Parallelism,
-		work:        make(chan Work, 100),
 	}
 }
 
 func (t *Parallel) Dir() string {
 	return t.compiler.Dir()
 }
 
-func (t *Parallel) Cleanup() {
```

Comment:
There is no such thing as "cleaning" a channel in Go. A channel is not a resource like a file that needs to have its resources released after use. A closed channel has different semantics than an open channel, but its not a step needed for "cleanup".